Week1/homework/js-exercises/3rd exercise/lognumber.js

```diff
@@ -0,0 +1,7 @@
+var numberx;
+console.log("I think this number is 9");
+console.log(numberx);
+var numberx = 9;
+console.log(numberx);
```

Comment:
Ah. Couple things here:
1. I am not sure if you are referring to any online material for JavaScript but it might be that it is a bit obsolete. This is because using `var` instead of `let` or `const` as we learnt in Week1 is the old way of doing things. Please try to use `let` or `const` instead of `var` now onwards.
2. Once you have declared variable `numberx` using ` keyword, you need not re-declare the variable. This is because variable declaration should happen only once in the program. If you wish to provide a different value to it, you should use variable re-assignment.